ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -76,48 +68,19 @@ export class ImportedDocuments extends React.Component {
     this.lastIndex = 0;
```

Comment:
```
this.mode = '';
this.status = '';
this.lastIndex = 0;
```
these 3 variables should not be stored as class attribute, I know it was there before but could you change it to be stored in the component state ?